content/articles/2019-10-styler-1.2.0.Rmarkdown

```diff
@@ -0,0 +1,228 @@
+
+The customization of the styling does not affect the command-line API
+(`styler::style_text()` and friends). We are not sure how users could best
+customize styling, but you can track progress on that in
+[r-lib/styler#319](https://github.com/r-lib/styler/issues/319).
+
+Remember that you can also set the environment variable `save_after_styling` to
+`TRUE` if you are tired of saving the file after styling it with the Addin.
+
+# Braces in function calls
+
+`tryCatch()` expressions often look like this:
+
+```{r}
+tryCatch(
+  {
+    exp(x)
+  },
+  error = function(x) x
+)
+```
+
+Prior to version 1.2.0, styler would return this odd formatting:
+
+```{r}
+tryCatch({
+  exp(x)
+},
+error = function(x) x
+)
+```
+
+Now, the line is broken before the opening curly brace in function calls, except
+if there is only one brace expression and it's the last in the function call.
+The typical use case is `testthat::test_that(...)`, i.e. the following code
+won't be modified:
+
+```{r}
+test_that("some condition holds", {
+  some_code()
+})
+```
+
+# Other changes
+
+Some other changes are:
+
+* styler depends on tibble >= 1.4.2 and runs 2x as fast as initially.
+
+* styler can style roxygen code examples in the source code of
+  packages.
+
+* styler can style `.Rnw` files.
+
+* The print method for the output of `style_text()` returns
+  syntax-highlighted code by default, controllable via the option
+  `styler.colored_print.vertical`.
+
+# Adaption of styler
+
+We'd like to highlight that styler integrates with various other tools you might
+be using:
+
+- As a git pre-commit hook. Two standard calls from the R console, and you are
+  all set. We are convinced that this is the preferred way of using styler to
+  ensure all your files are consistently formatted. Check out the [online
+  documentation](https://lorenzwalthert.github.io/precommit/) of the R package
+  `precommit` that also implements many other useful hooks.
+
+- `usethis::use_tidy_style()` styles your project according to the tidyverse
+  style guide.
+
+- `knitr::knitr()` and friends recognize the R code chunk option `tidy =
+  "styler"` for `.Rnw` and `.Rmd` files to pretty-print code.
+
+- `reprex::reprex(..., style = TRUE)` to prettify reprex code before printing.
+  To permanently use `style = TRUE` without specifying it every time, you can
+  add the following line to your `.Rprofile` (e.g. via
+  `usethis::edit_r_profile()`): `options(reprex.styler = TRUE)`.
+
+- There are plugins for
+  [Emacs](https://github.com/lassik/emacs-format-all-the-code) and
+  [VIM](https://github.com/w0rp/ale/pull/2401#issuecomment-485942966).
+
+# Outlook
+
+We have some cool new features in the pipeline such as
+[caching](https://github.com/r-lib/styler/pull/538) for faster styling or 
+[making styler ignore some lines](https://github.com/r-lib/styler/pull/560), 
+which you can try out by installing from the respective branches. We want to thank 
```

Comment:
Typically, we do the thank you in its own section that gets a header `### Acknowledgements`